packages/vite/src/node/server/index.ts

```diff
@@ -251,17 +246,9 @@ export interface ViteDevServer {
   watcher: FSWatcher
   /**
    * web socket server with `send(payload)` method
-   * @deprecated use `hot` instead
-   */
-  ws: WebSocketServer
-  /**
-   * HMR broadcaster that can be used to send custom HMR messages to the client
-   *
-   * Always sends a message to at least a WebSocket client. Any third party can
-   * add a channel to the broadcaster to process messages
    * @deprecated use `environment.hot` instead
    */
-  hot: HMRBroadcaster
```

Comment:
I don't think we can just remove it without a deprecation first. It was released as stable and we recommended people use it instead of `ws`, so people did

If it was marked as experimental or internal, I would understand the removal in v6, but I think we aim for a very backwards compatible version